### DISCUSSION TITLE
Handle stream even in uncertain network connection/disconnection

### DISCUSSION THREAD

**Discussion by anand-OSM** [HaishinKit/HaishinKit.swift#1556] [Q&A]
I am streaming video to the stream server. I just want to know if its able to handle such a situation. If network disconnect it holds stream in buffer and once it connect to stream server again, send buffer stream what was held in the offline along with current stream.

**Comment by shogo4405** [comment]
Unfortunately, that’s not how it’s implemented. When the connection is lost, the buffer is discarded. I think it would be difficult to implement that.

**Comment by anand-OSM** [comment]
So it doesn't handle stream if the network connection disconnects and reconnects, it drops the packets ?

**Reply by shogo4405** [reply]
Yes, the live streaming data is lost for the duration of time it takes to reconnect.
